packages/pn-personafisica-webapp/src/component/Contacts/__test__/DigitalContactsCodeVerification.context.test.tsx

```diff
@@ -2,24 +2,35 @@ import { ReactNode } from 'react';
 import * as redux from 'react-redux';
 import { fireEvent, RenderResult, screen, waitFor } from '@testing-library/react';
 
-import { LegalChannelType } from '../../../models/contacts';
+import { CourtesyChannelType, LegalChannelType } from '../../../models/contacts';
 import { render } from '../../../__test__/test-utils';
 import * as actions from '../../../redux/contact/actions';
+import * as hooks from '../../../redux/hooks';
 import {
   DigitalContactsCodeVerificationProvider,
   useDigitalContactsCodeVerificationContext,
 } from '../DigitalContactsCodeVerification.context';
 
 jest.mock('react-i18next', () => ({
   // this mock makes sure any components using the translate hook can use it without a warning being shown
-  useTranslation: () => {
-    return {
+  useTranslation: () => ({
       t: (str: string) => str,
-    };
-  },
-  Trans: (props: {i18nKey: string}) => props.i18nKey,
+    }),
+    Trans: (props: {i18nKey: string}) => props.i18nKey,
 }));
 
+const mockedStore = {
```

Comment:
Si potrebbe utilizzare il digitalAddresses esportato in test-utils.ts e utilizzato nei test del reducer